src/v2/Apps/Order/Utils/getStatusCopy.tsx

```diff
@@ -276,6 +276,15 @@ export const covidNote = (): React.ReactNode => {
   )
 }
 
+export const continueToInboxText = (): React.ReactNode => {
```

Comment:
I don't think putting the spacer here it's a good idea, despite my previous comment this function should return only the text and the component which uses it should place the Spacer there.

Reply:
Agreed.